RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 - [KOGITO-4455](https://issues.redhat.com/browse/KOGITO-4455) - Make Kogito Images/CLI to build a Kogito Workflow project from a workflow definition file
 - [KOGITO-4722](https://issues.redhat.com/browse/KOGITO-4722) - Support Job Service MongoDB distribution provisioning in Operator
 - [KOGITO-5315](https://issues.redhat.com/browse/KOGITO-5315) - Jobs service use unsupported authentication mechanism (part of Infinispan 12 upgrade - https://issues.redhat.com/browse/KOGITO-4341)
```

Comment:
These issues here were already solved, and it's on branch 1.8.x, correct? I think you can remove them.

Reply:
done